extensions/amp-ad-exit/0.1/test/test-amp-ad-exit.js

```diff
@@ -139,6 +156,13 @@ describes.realWin('amp-ad-exit', {
     win = env.win;
     toggleExperiment(win, 'amp-ad-exit', true);
     addAdDiv();
+    win.top.document.body.getResourceId = function() { return '6789'; };
```

Comment:
Any particular reason not to use `... = () => '6789';`?

Reply:
Nope...no reason other than I learned the "old way" and the "new way" isn't always instinctual yet. Will happily change it.